tests/rptest/services/redpanda.py

```diff
@@ -2778,7 +2778,7 @@ def search_log_all(self, pattern: str, nodes: list[ClusterNode] = None):
             # Match not found
             if exit_status != 0:
                 self.logger.debug(
-                    f"Did not find {pattern} on node {node.name}: {line}")
+                    f"Did not find {pattern} on node {node.name}")
```

Comment:
You may drop this commit. This change was merged in https://github.com/redpanda-data/redpanda/pull/9655

Reply:
if it's cherry picked then it'll probably get dropped in the merge automatically

Reply:
Cherry picked your commit onto my branch so I think git will figure it out